runtime/testy/stateManager.py

```diff
@@ -56,5 +56,9 @@ def start(self):
       elif cmd_type == SM_COMMANDS.HELLO:
         print("HELLO")
       # TODO: Add better error description
+      elif cm_type == SM_COMMANDS.SEND:
```

Comment:
I think the command `SEND` is too generally named since it specifically refers to sending to the UDPpackager.